concent_api/verifier/utils.py

```diff
@@ -354,3 +357,28 @@ def download_archives_from_storage(file_transfer_token, subtask_id, package_path
                 ErrorCode.VERIFIER_FILE_DOWNLOAD_FAILED,
                 subtask_id=subtask_id,
             )
+
+
+def parse_result_files_with_frames(frames, result_files_list, output_format):
+    parsed_files_to_compare = {}
+    for frame_number in frames:
+        for result_file_name in result_files_list:
+            if re.search(f'_{frame_number:>04}.{output_format.lower()}', result_file_name) is not None:
+                if not parsed_files_to_compare.get(output_format):
+                    parsed_files_to_compare.update({frame_number: [generate_verifier_storage_file_path(result_file_name)]})
```

Comment:
this can be straightforward: `parsed_files_to_compare[frame_number] = [generate_verifier_storage_file_path(result_file_name)]`

Reply:
Done